### PR TITLE
updated setup-konnectivity.md page

### PR DESCRIPTION
**Pull request by ashish-jaiswar**
### updated content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md
follow -up to [#30514 ](https://github.com/kubernetes/website/issues/30514)

### CODE REVIEW

File: content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md [hunk 1]
```diff
@@ -11,7 +11,8 @@ communication.
 
 ## {{% heading "prerequisites" %}}
 
-{{< include "task-tutorial-prereqs.md" >}}
+You need to have a Kubernetes cluster, and the kubectl command-line tool must
+be configured to communicate with your cluster. It is recommended to run this tutorial on a cluster with at least two nodes that are not acting as control plane hosts. If you do not already have a cluster, you can create one by using [minikube](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/).
```
Comment (mehabhalodiya) [inline content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md:15]:
```suggestion
You need to have a Kubernetes cluster, and the kubectl command-line tool must
be configured to communicate with your cluster. It is recommended to run this 
tutorial on a cluster with at least two nodes that are not acting as control 
plane hosts. If you do not already have a cluster, you can create one by using 
[minikube](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/).
```

Comment (mehabhalodiya) [reply content/en/docs/tasks/extend-kubernetes/setup-konnectivity.md:15]: I recommend doing these changes.

**Comment by ashish-jaiswar** [comment]
@bradtopol , @tengqm can you merge this PR ?

**Comment by ashish-jaiswar** [comment]
@divya-mohan0209 can you merge this PR?

**Comment by divya-mohan0209** [comment]
Will await approval from another sig docs reviewer.

**Review by mehabhalodiya — APPROVED** [review]
LGTM from SIG Docs!

**Comment by divya-mohan0209** [comment]
Taking lgtm from @mehabhalodiya [comment](https://github.com/kubernetes/website/pull/30664#pullrequestreview-819835207)
/approve

**Pull request merged**